pages/community/index.js

```diff
@@ -0,0 +1,529 @@
+/* eslint-disable react/no-unescaped-entities */
+import React from 'react';
+import Heading from '../../components/typography/Heading';
+import Button from '../../components/buttons/Button';
+import CommunityLayout from '../../components/layout/CommunityLayout';
+import IconRocket from '../../components/icons/Rocket';
+
+const orbit1 = [
```

Comment:
I think it's better we have this orbit data in another file and from there we import it to this file. Also, I see that you provided an `id` parameter to each element in array, which I think you can easily get from the map function (you are using). You can remove this from here and just add a `index` parameter in the map function.